packages/ngx-forms/src/lib/image-select/components/image-select/image-select.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, forwardRef, Input, OnInit } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { ImageSelectChoice } from '../../types/image-select.types';
+
+@Component({
+  selector: 'aui-image-select',
+  templateUrl: './image-select.component.html',
+  styleUrls: [
+    './image-select.component.scss',
+  ],
+  providers: [{
+    provide: NG_VALUE_ACCESSOR,
+    useExisting: forwardRef(() => ImageSelectComponent),
+    multi: true,
+  }],
+})
+export class ImageSelectComponent implements ControlValueAccessor, OnInit {
+  @Input() public choices: ImageSelectChoice[];
+  @Input() public maxSelectable: number | undefined;
+
+  public selectedImageKeys: string[] = [];
+  public isDisabled = false;
+
+  public ngOnInit(): void {
+    if (this.maxSelectable === undefined) {
+      this.maxSelectable = this.choices.length;
+    }
+  }
+
+  public updateModel: (_) => any = () => {
+  }
+
+  public registerOnChange(onChange: (_) => any): void {
+    this.updateModel = onChange;
+  }
+
+  public registerOnTouched(): void {
+  }
+
+  public setDisabledState(isDisabled: boolean): void {
+    this.isDisabled = isDisabled;
```

Comment:
Ik zie hier nergens styling voor, dus er is nu volgens mij geen onderscheid met een enabled optie.

Reply:
Css toegevoegd zodat bij disabled state een standaard cursor wordt getoond en geen 'pointer'.